RELEASING.md

```diff
@@ -0,0 +1,24 @@
+# Releasing
+
+This is released from the `master` branch unless it is a version older than `7.0.0`. If there is no 
+branch for the release that needs patching, create it from the tag.
+
+## Cutting the release
+
+### Requires contributor access
+
+- Check the [draft release notes](https://github.com/playframework/play-ebean/releases) to see if everything is there
```

Comment:
You should also configure the release drafter as GitHub action.
See for example https://github.com/playframework/twirl/tree/master/.github

Reply:
As long as this repo doesn't use sbt-dynver, a note to updated `version.sbt` might be good.